micro-ui/web/micro-ui-internals/packages/react-components/docker/Dockerfile

```
@@ -1,16 +1,17 @@
 WORKDIR /app
-ENV NODE_OPTIONS "--max-old-space-size=1792"
+ENV NODE_OPTIONS="--max-old-space-size=1792"
 
 COPY ${WORK_DIR} .
 
 #RUN node web/envs.js
-RUN node envs.js \
-    && yarn install \
-    && yarn build-storybook
+RUN node envs.js
+RUN yarn config set network-timeout 600000
```

Hadolint (2.12.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)